File: CPlusPlus/412.FizzBuzz.cpp
class Solution {
public:
    vector<string> fizzBuzz(int n) {
        vector<string> ret(n);
        for (int i = 1; i <= n; ++i) {
            bool silent = false;

            if (!(i % 3)) {
                silent = true;
                ret[i-1] += "Fizz";
            }
            if (!(i % 5)) {
                silent = true;
                ret[i-1] += "Buzz";
            }
            if (!silent)
                ret[i-1] = to_string(i);
        }
        return ret;
    }
};

File: Javascript/412.FizzBuzz.js
/**
 * @param {number} n
 * @return {string[]}
 */
var fizzBuzz = function(n) {
  var ret = [];
  for (let i = 1; i <= n; ++i) {
    if(!(i%15))
      ret.push('FizzBuzz');
    else if(!(i%3))
      ret.push('Fizz');
    else if(!(i%5))
      ret.push('Buzz');
    else
      ret.push(''+i)
  }
  return ret;
};